test/server/cards/01_SOR/events/AttackPatternDelta.spec.ts

```diff
@@ -0,0 +1,78 @@
+describe('Attack Pattern Delta', function() {
+    integration(function(contextRef) {
+        describe('Attack Pattern Delta\'s ability', function() {
+            beforeEach(function () {
+                contextRef.setupTest({
+                    phase: 'action',
+                    player1: {
+                        hand: ['attack-pattern-delta'],
+                        groundArena: ['battlefield-marine', 'wampa'],
+                        spaceArena: ['red-three#unstoppable']
+                    },
+                    player2: {
+                        leader: { card: 'luke-skywalker#faithful-friend', deployed: true },
+                    }
+                });
+            });
+
+            it(
```

Comment:
nit: the test name gets concatenated together with the describe name to form a sentence, so typically the test case name starts with lowercase "should". If you want to inspect the result you can just force the test to fail and you'll see the name

Reply:
Fixed.